dev-env-setup.md

```diff
@@ -22,24 +23,25 @@ This installation method removes existing Go installations, installs Go, and set
 
 ## Add the Go bin directory to your PATH
 
-Ensure the Go environment variables are [set properly](https://golang.org/doc/gopath_code#GOPATH) on your system. Many of the initial problems are related to incorrect environment variables.
+Ensure the Go environment variables are [set properly](https://golang.org/doc/gopath_code#GOPATH) on your system. Many
+of the initial problems are related to incorrect environment variables.
 
 1. Edit your `~/.bashrc` file and add `export PATH=$PATH:$(go env GOPATH)/bin`.
 2. To apply the changes, run `source ~/.bashrc`.
 
 ## Remove existing Ignite CLI installations
 
-Before you install a new version of Ignite CLI, remove all existing  installations.
+Before you install a new version of Ignite CLI, remove all existing installations.
 
 1. Remove the Ignite CLI binary with `rm $(which ignite)`
 
-   Depending on your user permissions, run the command with or without `sudo`.
+    Depending on your user permissions, run the command with or without `sudo`.
 
 2. Repeat this step until all Ignite CLI installations are removed from your system.
 
-`curl https://get.ignite.com/cli! | bash`
+`curl https://get.ignite.com/cli@v0.25.0! | bash`
```

Comment:
Why not keeping `latest` which returns the latest release ?

Reply:
I would add that if you look for `get.ignite.com` in the documentation, a lot of times it's mentioned w/o `@version`.

Reply:
I remember previously discussing with someone on the team that we wanted to include the version.  If my understanding is correct, `latest` just downloads `develop` and we really would rather push people to use our official tagged releases.

Reply:
>  If my understanding is correct, latest just downloads develop 

No, it downloads the latest github release (excluding pre-release, so excluding the nightly).

If you look at the code of `get.ignite.com` [here](https://github.com/ignite-hq/installer/blob/master/handler/handler_github.go#L64-L67), when no `@release` is passed, the github API is requested with this URL https://api.github.com/repos/ignite/cli/releases/latest, which as you can test, returns currently the v0.25.1, so the latest release.

Final proof :
```sh
$ curl https://get.ignite.com/cli |bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3962    0  3962    0     0  13657      0 --:--:-- --:--:-- --:--:-- 13709
Downloading ignite v0.25.1.....
######################################################################## 100.0%
Downloaded to /home/tom/src/ignite/installer/ignite

$ ./ignite version
Ignite CLI version:	v0.25.1
Ignite CLI build date:	2022-10-20T15:52:00Z
Ignite CLI source hash:	cc393a9b59a8792b256432fafb472e5ac0738f7c
Cosmos SDK version:	v0.46.3
Your OS:		linux
Your arch:		amd64
Your Node.js version:	v18.11.0
Your go version:	go version go1.19.2 linux/amd64
Your uname -a:		Linux 1gn-t12e 6.0.2-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 15 Oct 2022 14:00:49 +0000 x86_64 GNU/Linux
Your cwd:		/home/tom/src/ignite/installer
Is on Gitpod:		false
```